CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(QuickTurns VERSION 0.1 LANGUAGES CXX)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Widgets)
```

Comment:
Where does Qt come from? Is it a requirement to install Qt Framework from https://www.qt.io/download ? (In that case hardly we can run GitHub build actions)

Reply:
Probably we need to investigate this further as the project fails building without that. It was auto-generated by the QT creator.

Reply:
Yes we need `find_package(Qt...)` otherwise build will fail as we use Qt components.
Adding requirements/configuration/build steps will shed some light.